src/Repository/ProgrammeRepository.php

```diff
@@ -107,4 +107,14 @@ public function removeTrainerWithIdFromProgrammes(int $id): void
             ->getQuery()
             ->execute();
     }
+
+    public function returnBusiestDate(): array
+    {
+        $conn = $this->_em->getConnection();
+        $sql = 'SELECT DATE(start_date) as day, HOUR(start_date) AS hour, COUNT(programmes_customers.user_id) AS number 
```

Comment:
This query will give you all the hours in a day, and their number of participants.

What was needed was only the most busy hour for each day (no all hours like your query produces)

Reply:
I have understood. In my query I have most busy hours no meter if a day repeats... days should be distinct.... I'll review.

Reply:
Reviewed after your presentation